forge-gui-mobile/src/forge/adventure/stage/GameHUD.java

```diff
@@ -734,7 +736,7 @@ public void updateMusic() {
                 changeBGM(MusicPlaylist.WHITE);
                 break;
             case "waste":
-                changeBGM(MusicPlaylist.MENUS);
+                changeBGM(MusicPlaylist.COLORLESS);
```

Comment:
why not call MusicPlaylist.MENUS to play menu1.mp3 if that's the default music for waste?

Reply:
Because I want to be able to put music in a folder for colorless that's not the same as the menu music. Of course, the music I'm putting there isn't music I can upload to an open project like this, but having the distinction at least allows users to do this. I don't know where you guys get the music from, but it'd be a good idea to find a track to put here.